### PR TITLE
added date info in chart tooltip

### PR DESCRIPTION
**Pull request by PreetiW**
### Description

**Jira:** https://issues.redhat.com/browse/RHIROS-690

### Screenshot

image

image

### CODE REVIEW

File: src/Components/HistoricalDataChart/HistoricalDataChart.js [hunk 1]
```diff
@@ -119,7 +121,22 @@ export const HistoricalDataChart = ({ inventoryId }) => {
                         containerComponent={
                             <VictoryZoomVoronoiContainer
                                 labels={({ datum }) => {
-                                    return datum.childName.includes('scatter-') && datum.y !== null ? `${datum.name}: ${datum.y}%` : null;}
+                                    const isToday = new Date().toDateString() === new Date(datum.x).toDateString();
+                                    let xAxisDate = isToday ? 'Today' : `${new Date(datum.x).getDate()} ${MONTHS[new Date(datum.x).getMonth()]}`;
```
Comment (kgaikwad) [inline src/Components/HistoricalDataChart/HistoricalDataChart.js:125]: @PreetiW,
Here, you have created date object three times by passing the same value i.e. `datum.x`.
Though `new Date(datum.x)` objects are garbage collected automatically, will it be good to capture value in a variable & reuse it wherever required?

**Review by kgaikwad — CHANGES_REQUESTED** [review]

**Review by PreetiW — COMMENTED** [review]
Based on discussion with Stefan in past, need to update the the tooltip similar to below mockup.

image

**Review by PreetiW — COMMENTED** [review]
Based on our discussion in UX sync, I have created a seperate MR to address style related changes. Here is the link for same: https://issues.redhat.com/browse/RHIROS-927

File: src/Components/HistoricalDataChart/HistoricalDataChart.js (hunk 1 above)
Comment (PreetiW) [reply src/Components/HistoricalDataChart/HistoricalDataChart.js]: good catch @kgaikwad I have updated the PR by creating a single variable and pushed the changes.

File: src/Components/HistoricalDataChart/HistoricalDataChart.js [hunk 2]
```diff
@@ -119,7 +121,22 @@ export const HistoricalDataChart = ({ inventoryId }) => {
                         containerComponent={
                             <VictoryZoomVoronoiContainer
                                 labels={({ datum }) => {
-                                    return datum.childName.includes('scatter-') && datum.y !== null ? `${datum.name}: ${datum.y}%` : null;}
+                                    const isToday = new Date().toDateString() === new Date(datum.x).toDateString();
```
[the remark this answers is not in the document]
Comment (PreetiW) [reply src/Components/HistoricalDataChart/HistoricalDataChart.js]: @kgaikwad done and pushed the changes to PR for same 💯

**Review by kgaikwad — APPROVED** [review]
Recently updated change looks good to me. Thank you @PreetiW for addressing all review comments.

**Pull request merged**